src/main/java/core/basesyntax/Application.java

```diff
@@ -4,8 +4,8 @@ public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
         Lottery lottery = new Lottery();
-        System.out.println(lottery.getRandomBall().toString());
-        System.out.println(lottery.getRandomBall().toString());
-        System.out.println(lottery.getRandomBall().toString());
+        for (int i = 0; i < 3; i++) {
+            System.out.println(lottery.getRandomBall().toString());
```

Comment:
`toString()` call is not necessary, because sout recieves a param of type string and will make that call automatically